Tests/SwiftParserTest/Expressions.swift

```diff
@@ -90,6 +90,45 @@ final class ExpressionTests: XCTestCase {
   }
 
   func testKeypathExpression() {
+    AssertParse(
+      #"""
+      \.?.foo
+      """#,
+      substructure: Syntax(
+        CodeBlockItemListSyntax([
+          CodeBlockItemSyntax(
+            item: Syntax(
+              KeyPathExprSyntax(
```

Comment:
IMO it would have been sufficient to start the `substructure` with `KeyPathExprSyntax` because the wrapping `CodeBlockItem` isn’t really interesting. But it doesn’t really do any harm either.

Reply:
Okay, that makes sense.